integr8ly/tutorial-web-app
# INTLY-997: Add github repo and last commit date to walkthrough overview

## Motivation
See https://issues.jboss.org/browse/INTLY-997... requirement was to add metadata on the first (overview) page of a walkthrough that links to the walkthrough source on github and also displays the date it was last updated.

# What
Create a new component that will display the github repo link and last updated date to the overview page. If the github repo is ours, the link label will display 'Red Hat', otherwise it will display 'Community'.

## Verification Steps
1. Go to the first page of a walkthrough (the overview page... tutorial.js).
2. On the right side, above 'Resources', there should be a new section named 'Details', and both sections should be under a new heading named 'About this walkthrough'.
3. Verify that the Source: field displays 'Red Hat' if viewing our default walkthroughs, and 'Community' if viewing a custom (any other) walkthrough.
4. Verify that when the 'Red Hat' or 'Community' link is clicked, the github repo for that source is opened in a new browser tab.
5. Verify that the last updated date is valid and formatted correctly.

## Checklist:

## Progress

## Additional Notes
Screen cap:
![wt_metadata](https://user-images.githubusercontent.com/39063664/54728550-f322f000-4b54-11e9-9f73-5a5101cdd54c.png)
